ansible/vars/dpr.yml

```diff
@@ -0,0 +1,28 @@
+ec2_key_name: perf-tests
+profile: default
+mfa_sn: arn:aws:iam::225542912595:mfa/christopher.hayes
+mfa: 923996
```

Comment:
Does this mean it is specifically tied to your MFA device/profile? In the future will we have to change this at all? If so, is that documented anywhere?

Reply:
I actually don't think that's doing anything. I'll clean it up in the next PR waiting for this to merge.